main.go

```diff
@@ -175,11 +180,38 @@ func handle(w http.ResponseWriter, r *http.Request) {
 		defer f.Close()
 		http.ServeContent(w, r, "main.wasm", time.Now(), f)
 		return
+
+	case "__wait":
+		waitForUpdate(w, r)
+		return
+	case "__notify":
```

Comment:
Who fetches `__notify`?

Reply:
Ah, we expect we call this explicitly?

Reply:
Yeah, the user would invoke this explicitly.